### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by SheepReaper**
implements #3

- [x] Determine what the report abuse email should be

### CODE REVIEW

**Comment by SheepReaper** [comment]
I filled in lfs-support@lists.linuxfromscratch.org, is there a better contact email to use for reports of abuse?

**Comment by pierre-labastie** [comment]
> I filled in [lfs-support@lists.linuxfromscratch.org](mailto:lfs-support@lists.linuxfromscratch.org), is there a better contact email to use for reports of abuse?

Hmm: mail addresses at lists.linuxfromscratch.org have well defined purposes. See the descriptions [there](http://www.linuxfromscratch.org/mail.html). I do not know whether we could create a specific address. But I think alfs-discuss@lists.linuxfromscratch.org would be more adapted in the absence of a more specific address.

**Comment by pierre-labastie** [comment]
> > I filled in [lfs-support@lists.linuxfromscratch.org](mailto:lfs-support@lists.linuxfromscratch.org), is there a better contact email to use for reports of abuse?

Actually, the reporter name should remain confidential, we cannot use a public mailing list...
The mail should be only sent to us; and shouldn't be readable by any other person. Is it possible to create a collective address for us three?

**Comment by SheepReaper** [comment]
If by "collective address" you mean a private distribution list, you can
create a new listserv group that you cannot join or must be approved to
join. I don't know if listserv also has the ability to just email the
admins of a list, such as alfs-discuss.admins@lists.linuxfromscratch.org.

I'm also assuming you're using listserv, I don't know what you're actually
using.

An alternative would be to start a private Google group:
alfs-abuse@googlegroups.com

Unless you know of a free way to create private distribution lists.

GitHub enterprise lets you assign email addresses to teams, I don't know if
that's a feature of GitHub free...

**Comment by pierre-labastie** [comment]
The only list administrator at linuxfromscratch.org is Bruce. We may ask him to create a list. But I'm not sure he can create a list which accepts emails without subscription... And then transfers it to only a few person

I do not know a free way to create a private distribution list. I fear that a private Google group would accept mails only from members...

Hmm, maybe, what we can do is create a google account, something like alfs-abuse<span>@</span>gmail.com, and share the password between the three of us, and no one else. Then tune our mail readers so as to leave mail on server.

**Comment by SheepReaper** [comment]
I sent you an invite to the group: alfs-abuse

Memebrship is by invitation only, anyone on the web can send a message to
it, and messages are private.

Please help me test to see if this meets the requirements.

**Comment by pierre-labastie** [comment]
> I sent you an invite to the group: alfs-abuse Memebrship is by invitation only, anyone on the web can send a message to it, and messages are private. Please help me test to see if this meets the requirements.

Hi Bryan, could you please send it to pierrot.labastie@gmail.com? This is what I use for my gmail account, and I seem to be unable to accept the invitation from the link in the mail sent to pierre.labastie@neuf.fr.

**Review by jhuntwork — APPROVED** [review]
Seems good to me

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at lfs-support@lists.linuxfromscratch.org. All
```
Comment (pierre-labastie) [inline CODE_OF_CONDUCT.md]: The mail address should be changed,now that we have one.

**Comment by SheepReaper** [comment]
does anyone have a secondary, non-gmail account that they can send a test
message to alfs-abuse@googlegroups.com ? I just want to verify that every
message sent from a non-Owner or non-Manager is only visible by the one who
sent it and by the group managers/owners only.

Jeremy, you have not accepted your invitation to the group, however, I
cannot assign Manager/Owner role to a non-gmail account. Do you have gmail
alias I can invite you to?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (SheepReaper) [reply CODE_OF_CONDUCT.md]: updated in https://github.com/automate-lfs/jhalfs/pull/7/commits/001888aaaa0aaa2

**Comment by pierre-labastie** [comment]
> does anyone have a secondary, non-gmail account that they can send a test message to alfs-abuse@googlegroups.com ?

Done, seems to work. Will approve. Please do the merge.

**Review by pierre-labastie — APPROVED** [review]
Please merge.

**Comment by jhuntwork** [comment]
@bryan5989 - Sorry, I missed the email and was a little busy. Can you re-send the invite? My email account is a gmail account.

**Comment by SheepReaper** [comment]
Done

**Pull request merged**